XamarinCommunityToolkit/Effects/RemoveBorder/RemoveBorderEffect.android.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Graphics.Drawables.Shapes;
+using Android.Runtime;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using Effects = Xamarin.CommunityToolkit.Android.Effects;
+
+[assembly: ExportEffect(typeof(Effects.RemoveBorderEffect), nameof(RemoveBorderEffect))]
+
+namespace Xamarin.CommunityToolkit.Android.Effects
+{
+	public class RemoveBorderEffect : PlatformEffect
+	{
+		Drawable originalBackground;
```

Comment:
I saw that on iOS you use the `?`, makes sense to use here as well?
```suggestion
		Drawable? originalBackground;
```

Reply:
In the iOS renderer the old style `UITextBorderStyle` is an enum so needs to be explicitly marked as nullable. Marking the `Drawable` as nullable is not necessary as it already can be `null` 😊

Reply:
Got ya, I thought that you were using the new nullable feature (: